diff --git a/node_modules/wav/lib/file-writer.js b/node_modules/wav/lib/file-writer.js
index 451a84f..e543632 100644
--- a/node_modules/wav/lib/file-writer.js
+++ b/node_modules/wav/lib/file-writer.js
@@ -3,7 +3,7 @@
  * file, and is also a readable stream that will output the final WAV file.
  */
 
-var fs = require('fs');
+// var fs = require('fs');
 var Writer = require('./writer');
 var inherits = require('util').inherits;
 
@@ -34,7 +34,8 @@
  * the WAV file to be written to.
  */
 function FileWriter (path, opts) {
-  var stream = fs.createWriteStream(path, opts);
+  // var stream = fs.createWriteStream(path, opts);
+  var stream = {};
   Writer.call(this, opts);
   this.path = path;
   this.stream = stream;
diff --git a/node_modules/wav/lib/reader.js b/node_modules/wav/lib/reader.js
index 720ea21..bb07127 100644
--- a/node_modules/wav/lib/reader.js
+++ b/node_modules/wav/lib/reader.js
@@ -3,7 +3,7 @@
  * file.
  */
 
-var fs = require('fs');
+// var fs = require('fs');
 var util = require('util');
 var Stream = require('stream');
 var constants = require('./constants');
@@ -32,7 +32,8 @@
  * the WAV file to be read from.
  */
 function Reader (path, opts) {
-  var stream = fs.createReadStream(path, opts);
+  // var stream = fs.createReadStream(path, opts);
+  var stream = {};
   Stream.call(this, opts);
   this.path = path;
   this.stream = stream;
